website/docs/sdk-reference/dart.md

```diff
@@ -71,6 +71,8 @@ ConfigCatClient.close(client: client); // closes a specific client
 | `mode`                      | Optional, sets the polling mode for the client. [More about polling modes](#polling-modes). |
 | `logger`                    | Optional, sets the internal logger and log level. [More about logging](#logging). |
 | `override`                  | Optional, configures local feature flag & setting overrides. [More about feature flag overrides](#flag-overrides). |
+| `defaultUser`               | Optional, configures the default user. [More about default user](#default-user). |
```

Comment:
please try to avoid the word `configure` and `configuration`

Reply:
done